text/survey_analysis_report2.md

```diff
@@ -0,0 +1,191 @@
+---
+title: "Towards a standardized research folder structure."
+bibliography:  "GIN-tonic-paper-zotero.bib"
+author:
+  - Julien Colomb
+  - The Gin-tonic team
+output: 
+  bookdown::html_document2:
+    keep_md: true
+
+---
+
+
+![titleimage: Some positive feedback received in the survey.](../50_figures/pulication_ready_figures/blogtitle.jpeg)
+
+# Summary
+
+In contrast to software developers, researchers mostly do not organize their files according to some common standard.
+While data managers recommend to design such an organisation,
+they fail at providing clear recommendations or examples;
+and there is no time specifically assigned to this task in the researcher's work.
+We believe that providing researchers with a commonly accepted folder tree structure template could make a huge difference in promoting data management and research collaboration.
+In this blog, we present the results of an initial survey run in three neuroscientific research consortia (LINK).
+
+
+We collected the feedback from 51 neuroscientists presented with two initial template drafts,
+analysed their responses,
+and built an updated template based on the feedback that is presented here.
+This new template takes three levels of data organisation into account (laboratory, project, and experiment),
+while staying fairly simple and flexible, especially when implemented it in a git workflow (using the GIN-tonic tool we are developing).
+Researchers from all research domains are now invited to review this draft in a second survey (run until mid January 2021, LINK).  
+
+
+
+# Introduction
+
+Every day, researchers spend time doing file management on their computers (creating, downloading, naming, moving, saving, copying, reviewing, navigating, searching for, sharing, and deleting files and folders).
+While many different initiatives and tools tried to improve file management (using tags, databases and search algorithms), the use of folder tree structure appeared to be unavoidable and necessary [@dinneenUbiquitousDigitalFile2020].
+In addition, both proponents of reproducible research and data management experts recommend researchers the use of an appropriate folder organizational structure ([@LibraryCarpentryLcfairresearch2019], [@theturingwaycommunityTuringWayHandbook2019]).
+However, only few actually provide examples or attempts to homogenize such a structure (see [@vuorreSharingOrganizingResearch2020] and [@wilsonGoodEnoughPractices2017] for exceptions).
```

Comment:
Is the word "to homogenize" common in the field? Can it be reworded to be one of:
- standardize
- disperse
- spread

Reply:
What about "make uniform" and what is wrong with homogenize in your opinion ?